docs/blockchain/contracts/system-contract/system-actions/delegatebw.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: 'delegatebw - stake tokens for POWER'
```

Comment:
what was the reason for mentioning those small descriptions? can they mention it in the body of the article? or is it useful?

Reply:
It's mentioned in the body of the article already. It was clogging up the navbar.